source/_docs/drush-versions.md

```diff
@@ -6,6 +6,8 @@ categories: [drupal]
 ---
 By default, Pantheon runs Drush 8 on newly created Drupal sites. Drush 8 is compatible with Drupal 6, 7, and 8.
 
+The Drush version is set when the site is created. Sites created a few years ago or more may use an older version of Drush that is incompatible with later versions of PHP. Updating Drush can be accomplished by [a simple addition to your pantheon.yml file](#configure-drush-version).
```

Comment:
Drush 8 became the default version of Drush for Drupal 8 sites on 3 Sept 2015.  It became the default for all Drupal sites on 4 Nov, 2015